### DISCUSSION TITLE
Integration of structured outputs.

### DISCUSSION THREAD

**Discussion by GalShaffirLibba** [RageAgainstThePixel/com.openai.unity#270] [Q&A]
Hey! 
Recently openAI introduced the structured outputs that forces the output not only to be in a valid JSON format but also forces it to be in a JSON format specified by the user. 
more about it here: 
https://openai.com/index/introducing-structured-outputs-in-the-api/

I was wondering if it is available to do in the current version of the plugin, if not are there any plans to integrate it in the future? 

once again, thanks for this great plugin!
Gal.

**Comment by StephenHodgson** [comment]
Thanks, I've already got a issue tracking this:

- https://github.com/RageAgainstThePixel/OpenAI-DotNet/issues/343
